bank_00b.asm

```diff
@@ -1,12 +1,3286 @@
 SECTION "ROM Bank $00b", ROMX[$4000], BANK[$b]
 
-INCBIN "baserom.gb", $2c000, $2d8b9 - $2c000
+;Byte 1 and 2 are some sort of X and Y offsets for the Metasprite.
+;Byte 3 is the tile used by the Metasprite.
+;Byte 4 is the attributes, listed below
+;7: Render priority
+;6: Y flip
+;5: X flip
+;4: End of Metasprite
```

Comment:
Thanks for doing this dump.

However, please amend this data structure description, as it's incorrect.  Each metasprite entry is simply a raw OAM entry, where bit 0 of the last byte marks the end of the metasprite.  https://bgb.bircd.org/pandocs.htm#vramspriteattributetableoam

Reply:
Sorry, I changed the bit for my own project and I guess I forgot to change it in the description.